src/translations/dynamic/menu-locale-fr.ts

```diff
@@ -11,24 +11,29 @@ const menu_locale_fr = {
     Disconnect3WTransformer: 'Déclencher le transformateur',
     DisconnectGenerator: 'Déclencher le group',
     DisconnectLine: 'Déclencher la ligne',
+    DisconnectHvdcLine: 'Déclencher la ligne Hvdc',
     DisconnectLoad: 'Déclencher la consommation',
     EnergiseOnOneEndLine: 'Mettre sous tension à vide depuis {substation}',
     Lockout2WTransformer: 'Consigner le transformateur',
     Lockout3WTransformer: 'Consigner le transformateur',
     LockoutLine: 'Consigner la ligne',
+    LockoutHvdcLine: 'Consigner la ligne Hvdc',
     NodeFaultBus: 'Court-circuit sur le bus',
     SwitchOnLine: 'Mettre en service la ligne',
     Trip2WTransformer: 'Déclencher le transformateur',
     Trip3WTransformer: 'Déclencher le transformateur',
     TripLine: 'Déclencher la ligne',
+    TripHvdcLine: 'Déclencher la Hvdc ligne',
```

Comment:
```suggestion
    TripHvdcLine: 'Déclencher la ligne HVDC',
```
partout pareil, et HVDC en majuscule c'est un acronyme